### PR TITLE
minor: add doc_home_display_name setting

### PR DESCRIPTION
**Pull request by houqp**
Also add documentation for sqlite3 binding.

Depends on https://github.com/koreader/koreader-base/pull/465

### CODE REVIEW

**Comment by houqp** [comment]
@baskerville, I changed the setting name here to better distinguish koreader source home and document home. Please let me know if you have any concern other than the ones we discussed in https://github.com/koreader/koreader/pull/2337.

**Comment by baskerville** [comment]
I'm fine with this, but please note that `home_dir` appears somewhere else in `frontend/apps/filemanager/filemanager.lua` and also in `reader.lua`.

And when using the *Set as HOME directory* entry of the long tap press dialog, the current path view should be updated.

**Comment by houqp** [comment]
OOPS, good catch. In this case, it will break other user's setting. Sorry, I thought this is a new setting. Not with my dev machine right now, I will update my PR tmr.

**Comment by Frenzie** [comment]
Don't forget my comments. :)

**Comment by baskerville** [comment]
@Frenzie I've already incorporated your suggestion: the substitution won't occur if `home_dir_display_name` isn't set.

**Comment by Frenzie** [comment]
@baskerville I meant @houqp on the totally unrelated stuff.

**Comment by houqp** [comment]
@Frenzie sorry, what comment were you referring to?

**Comment by Frenzie** [comment]
@houqp You can't see my three "review" comments? It's just a few insignificant language issues, nothing code-related, but it'd still be nice to get them in. I noticed you also ignored what I wrote on which I actually think carries a little more weight https://github.com/koreader/koreader/pull/2320

**Comment by houqp** [comment]
@Frenzie did you forget to click publish review? ;P

**Comment by houqp** [comment]
By default, all the reviews won't be visible to other people until you publish them. I was totally confused by this the first time I use it. Could you also comment your previous review in #2320 ? I didn't see anything there :(

**Comment by houqp** [comment]
When you publish the review (button at the top right), remember to click request change so this PR cannot be merged without an approval.

**Comment by Frenzie** [comment]
Ohhhh. I thought it was basically just a new, optional feature on the regular old comments I'm used to. This explains so much.

**Comment by houqp** [comment]
Cool, all fixed ;)

File: platform/kobo/release-ip.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 
+# PATH export is only needed if you run this script manually from a shell
```
Comment (Frenzie) [inline platform/kobo/release-ip.sh]: Ahhhh, so *that*'s what it's for. =)

**Pull request merged**